libs/community/langchain_community/retrievers/project_gutenberg.py

```diff
@@ -0,0 +1,76 @@
+from typing import List
```

Comment:
Could you introduce this as a utility? Most of the utilities can be thought of as clients. Then users can reuse the client to either create a custom tool, create a retriever or create a document loader

Reply:
Thanks for the feedback @eyurtsev